model/textparse/openmetricsparse_test.go

```diff
@@ -64,7 +64,48 @@ _metric_starting_with_underscore 1
 testmetric{_label_starting_with_underscore="foo"} 1
 testmetric{label="\"bar\""} 1
 # TYPE foo counter
-foo_total 17.0 1520879607.789 # {id="counter-test"} 5`
+foo_total 17.0 1520879607.789 # {id="counter-test"} 5
+foo_created 1000
+foo_total{a="b"} 17.0 1520879607.789 # {id="counter-test"} 5
+foo_created{a="b"} 1000
```

Comment:
Would you mind adding `# HELP` entries with the objective of documenting the test itself?
Like:

`# HELP foo Counter with and without labels to certify CT is parsed for both cases`

`# HELP bar Summary with CT at the end, making sure we find CT even if it's multiple lines a far`

`# HELP baz Histogram with the same objective as above's summary`

`# HELP something Histogram with _created before the buckets.`

and etc etc It could help future readers understand what exactly we're testing with each entry

Reply:
Done!